test/collins/controllers/IpmiApiSpec.scala

```diff
@@ -0,0 +1,30 @@
+package collins.controllers
+
+import collins._
+import org.specs2._
+import specification._
+import play.api.test.WithApplication
+import org.specs2.matcher.JsonMatchers
+
+class IpmiApiSpec extends mutable.Specification with ControllerSpec {
+
+  "IPMI API Specification".title
+
+  args(sequential = true)
+
+  "the REST API" should {
+    "Support getting ipmi pools" in new WithApplication with AssetApiHelper {
```

Comment:
The `WithApplication` can use something like `FakeApplication` and provide individual parameters? However, ive not had great experience overriding a config this way (i gave up overriding the IPAM pool configs :()